packages/core/src/llm-core/utils/similarity.ts

```
@@ -0,0 +1,141 @@
+
+class TextTokenizer {
+    private static isChinese(text: string): boolean {
+        return /[\u4e00-\u9fa5]/.test(text)
+    }
+
+    private static isJapanese(text: string): boolean {
+        return /[\u3040-\u30ff\u3400-\u4dbf\u4e00-\u9fff]/.test(text)
+    }
+
+    static tokenize(text: string): string[] {
+        if (this.isChinese(text)) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)